examples/gallery/maps/choropleth_map.py

```diff
@@ -14,45 +14,36 @@
 """
 
 # %%
-import geodatasets
 import geopandas as gpd
 import pygmt
 
-# Read the example dataset provided by geodatasets.
-gdf = gpd.read_file(geodatasets.get_path("geoda airbnb"))
-print(gdf.head())
+provider = "https://naciscdn.org/naturalearth/"
+world = gpd.read_file(f"{provider}50m/cultural/ne_50m_admin_0_countries.zip")
```

Comment:
I prefer a URL like this:
```suggestion
provider = "https://naciscdn.org/naturalearth"
world = gpd.read_file(f"{provider}/50m/cultural/ne_50m_admin_0_countries.zip")
```